sway/ipc-json.c

```diff
@@ -1000,6 +1001,20 @@ json_object *ipc_json_describe_input(struct sway_input_device *device) {
 					layout ? json_object_new_string(layout) : NULL);
 			}
 		}
+
+		json_object *modifiers_json = json_object_new_array();
+		uint32_t keyboard_modifiers = wlr_keyboard_get_modifiers(keyboard);
+		for (int i = 0; i < (int)(sizeof(modifiers) / sizeof(struct modifier_key)); i++) {
```

Comment:
Can we use `get_modifier_names` instead? That way we can also keep the `modifiers` global variable static in `sway/input/keyboard.c`.